resources/application-ide.properties

```diff
@@ -13,3 +13,11 @@ identity.key.password=changeit
 trust.store.password=changeit
 identity.store.alias=cid
 compress.output=true
+s3.bucket=demobucket
+s3.prefix.folder=hbase-to-mongo-export
+s3.manifest.bucket=demobucket
+s3.manifest.prefix.folder=export-manifests
+#s3.bucket=d69d3ea0-4d14-4a57-b017-4ec911a475b9
```

Comment:
Should remove the commented lines really

Reply:
Removed, can you re-review if you get a moment.